tests/TestCase/Filesystem/FileTest.php

```diff
@@ -503,7 +503,7 @@ public function testLastChange()
     /**
      * testWrite method
      *
-     * @return void
+     * @return bool|void
```

Comment:
Same issue here. Must stay void and instead the test should be skipped properly.